src/ott/utils.py

```diff
@@ -59,3 +60,56 @@ def is_jax_array(obj: Any) -> bool:
     # https://jax.readthedocs.io/en/latest/jax_array_migration.html
     return isinstance(obj, (jax.Array, jnp.DeviceArray))
   return isinstance(obj, jnp.DeviceArray)
+
+
+def default_progress_fn(
+    status: Tuple[np.ndarray, np.ndarray, np.ndarray, NamedTuple], *args
+) -> None:
+  """This default callback function reports progress by printing to the console.
+
+  Args:
+      status: tuple describing the current iteration number,
+        the number of inner iterations after which the error is computed,
+        the total number of iterations, each wrapped in a ``jax.Array``, and
+        the current Sinkhorn state.
+      args: unused, see <https://jax.readthedocs.io/en/latest/jax.experimental.host_callback.html>
+
+  This function updates the progress bar only when the error is computed
+  (every `solver.inner_iterations`, typically `10`).
+
+  Note: the user can provide a slightly modified version of this callback in
+  order to use a progress bar. To do so, replace the print statement with the
+  following:
+
+    ```
+    pbar.set_description_str(f"error: {error:.6f}")
+    pbar.total = total_iter
+    pbar.update()
+    ```
+
+  assuming `pbar` is defined at the callsite with the following code:
+
+    ```
+    with tqdm() as pbar:
+      out_sink = jax.jit(sinkhorn.Sinkhorn(progress_fn=progress_fn))(lin_problem)
+    ```
+
```

Comment:
Missing `Returns:` section

Reply:
This function returns nothing, do you want the following?

```rst
Returns:
  None
```